### ISSUE TITLE
Add election date requested to non-PII database

### ISSUE THREAD

**Issue by katherinepdavis** [BlueprintKansas/ksvotes.org#548]
When pulling the aggregate reporting data, we're reporting # of individuals who have requested a ballot rather than total # of ballot requests submitted (primary and general should each count).

**Comment by karpet** [comment]
I believe we support multiple election dates for AB prior to the primary.

Right now this is part of the export in the `r_elections` csv column. You're saying you want a flat `election_date` column on the database so that you can run queries that shard by election date? That might be tricky if there are multiple pipe-delimited dates.

**Comment by katherinepdavis** [comment]
Yes, we allow simultaneous submissions for primary and general. 

Ultimately the goal is to be able to show an accurate count of AB applications through our site. I think there are a number of potential approaches - have a abs_requested column that would show 0, 1, or 2 - then we could sum it. Or have a Boolean column for prim AB Submitted and another for gen AB Submitted. I'm sure there are other solutions too. I'm open to whatever is easiest to implement.

**Comment by penguinopus** [comment]
FYI,
I just redid my grep to find ksv entries that requested both:
grep "August 4, 2020" ksvotes-production_20201008.csv | grep "November 3,
2020" | wc
  27387 4819419 53169234

So 27387 extra ABs were requested, so we broke 100K!

   - bam
